internals/overlord/logstate/syslog/syslog.go

```diff
@@ -0,0 +1,300 @@
+package syslog
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"net/url"
+	"os"
+	"sort"
+	"time"
+
+	"github.com/canonical/pebble/internals/servicelog"
+)
+
+const (
+	defaultMaxRequestEntries    = 100
```

Comment:
The "default" prefix is accurate, however, I think that's implied given they're `const`, and I'd rather a shorter name that matches the `loki` and `opentelemetry` package version, so let's call remove the `default` prefixes here:
```suggestion
	maxRequestEntries    = 100
```